core/common/src/main/scala/org/typelevel/otel4s/context/propagation/TextMapPropagator.scala

```diff
@@ -84,31 +74,70 @@ trait TextMapPropagator[F[_], Ctx] {
     * @return
     *   a copy of the carrier, with new fields injected
     */
-  def injected[A: TextMapUpdater](ctx: Ctx, carrier: A): A
+  def inject[A: TextMapUpdater](ctx: Ctx, carrier: A): A
 }
 
 object TextMapPropagator {
 
-  def apply[F[_], Ctx](implicit
-      ev: TextMapPropagator[F, Ctx]
-  ): TextMapPropagator[F, Ctx] = ev
+  /** Creates a [[TextMapPropagator]] which delegates injection and extraction
+    * to the provided propagators.
+    *
+    * @example
+    *   {{{
+    * val w3cPropagator: TextMapPropagator[Context] = ???
+    * val httpTracePropagator: TextMapPropagator[Context] = ???
+    * val textMapPropagator = TextMapPropagator.composite(w3cPropagator, httpTracePropagator)
+    *   }}}
+    * @tparam Ctx
+    *   the context to use to extra or inject data
+    */
+  def composite[Ctx](
```

Comment:
there's something about the name `composite` that I really dislike, but I'm not sure what. maybe just the length? idk. what do you think of naming the method `of` or `apply`?

Reply:
The spec says we should allow the creation of a composite propagator, but says nothing about the method name :) 

https://opentelemetry.io/docs/specs/otel/context/api-propagators/#composite-propagator

I guess, `apply` would fit